packages/reports/jest.config.js

```diff
@@ -3,17 +3,17 @@ const baseConfigForPackages = require('../../jest.config.base');
 module.exports = {
     ...baseConfigForPackages,
     moduleNameMapper: {
-        "\\.s(c|a)ss$": "<rootDir>/../../__mocks__/styleMock.js",
-        "^.+\\.svg$": "<rootDir>/../../__mocks__/styleMock.js",
-        '^_common\/(.*)$': "<rootDir>/src/_common/$1",
-        '^App\/(.*)$': "<rootDir>/src/App/$1",
-        '^Assets\/(.*)$': "<rootDir>/src/Assets/$1",
-        '^Constants\/(.*)$': "<rootDir>/src/Constants/$1",
-        '^Constants$': "<rootDir>/src/Constants/index.js",
-        '^Documents\/(.*)$': "<rootDir>/src/Documents/$1",
-        '^Modules\/(.*)$': "<rootDir>/src/Modules/$1",
-        '^Utils\/(.*)$': "<rootDir>/src/Utils/$1",
-        '^Services\/(.*)$': "<rootDir>/src/Services/$1",
-        '^Stores\/(.*)$': "<rootDir>/src/Stores/$1",
+        '\\.s(c|a)ss$': '<rootDir>/../../__mocks__/styleMock.js',
+        '^.+\\.svg$': '<rootDir>/../../__mocks__/styleMock.js',
+        '^_common(.*)$': '<rootDir>/src/_common/$1',
+        '^App(.*)$': '<rootDir>/src/App/$1',
+        '^Assets(.*)$': '<rootDir>/src/Assets/$1',
```

Comment:
i'm not sure about these changes. are they work as before?

Reply:
no I removed extra slashes :)))) let me update that